com/madishetti/JedisManager.cfc

```diff
@@ -132,7 +138,12 @@ component accessors="true" {
             // Get a Jedis resource from the pool
             jedis = getJedisResource();
             // Retrieve data from the cache
-            return jedis.get( arguments.cacheKey );
+            var cacheData = jedis.get( arguments.cacheKey );
+            // Check if the cacheData is a json value
+            if(isJson(cacheData)){
+                cacheData = deserializeJson(cacheData);
```

Comment:
See above - using .toJson and then deserializeJson is a bit inconsistent in style, I think.

can fromJson() be used here? Or maybe use the classic functions in both places.